v3/src/components/graph/utilities/graph-utils.ts

```diff
@@ -101,15 +101,38 @@ export function getPointTipText(caseID: string, attributeIDs: string[], dataset?
 }
 
 export interface IMatchCirclesProps {
+  dataset?: IDataSet,
   caseIDs: string[]
   dotsElement: SVGGElement | null
   pointRadius: number
   enableAnimation: React.MutableRefObject<boolean>
   instanceId: string | undefined
 }
 
+export function handleClickOnDot(event: MouseEvent, caseID: string, dataset?: IDataSet) {
+  const [, caseId] = caseID.split("_"),
+    extendSelection = event.shiftKey,
```

Comment:
```suggestion
    extendSelection = event.shiftKey || event.metaKey,
```
Cmd/meta key should behave the same way.